ios-deploy.xcodeproj/project.pbxproj

```diff
@@ -226,6 +231,19 @@
 			shellPath = /bin/sh;
 			shellScript = "rm -rf \"${PROJECT_DIR}/_Frameworks\"";
 		};
+		7B757512209A0D1E00B885EB /* ShellScript */ = {
+			isa = PBXShellScriptBuildPhase;
+			buildActionMask = 2147483647;
+			files = (
+			);
+			inputPaths = (
+			);
+			outputPaths = (
+			);
+			runOnlyForDeploymentPostprocessing = 0;
+			shellPath = /bin/sh;
+			shellScript = "echo \"\\\"# AUTO-GENERATED - DO NOT MODIFY\\\\\\n\\\"\" > src/ios-deploy/lldb_cmds.h\nawk '{ print \"\\\"\"$0\"\\\\n\\\"\"}' src/scripts/lldb_cmds >> src/ios-deploy/lldb_cmds.h\n";
```

Comment:
I know that other people are building this not using xcodebuild and that this change will require updating those build systems to create a system to emulate what this awk script does (adding escape sequences to src/scripts/lldb_cmds and placing it in src/ios-deploy/lldb_cmds.h).